client/state/selectors/get-post-revision-changes.js

```diff
@@ -4,22 +4,53 @@
  * External dependencies
  */
 
-import { findIndex } from 'lodash';
+import { findIndex, get, isUndefined, map, reduce, omitBy } from 'lodash';
```

Comment:
Minor: It was mentioned to me the last time that we alphabetize these. Maybe move the `omitBy`.

Reply:
thanks. will do